src/measure_theory/measure/doubling.lean

```diff
@@ -104,6 +107,26 @@ begin
     exact ennreal.mul_le_mul (ennreal.coe_le_coe.2 (le_max_left _ _)) le_rfl }
 end
 
+lemma eventually_measure_le_scaling_constant_mul (K : ℝ) (hK : 0 < K) :
```

Comment:
Not sure why you need `hK` here. Normally, the definition of the scaling constant should guarantee this whatever `K`.